src/api/completions.ts

```diff
@@ -195,6 +179,15 @@ completions.delete('/:id', async (req, res) => {
       return;
     }
 
+    //check for deletion (I know this is bad but its a bandaid)
```

Comment:
I think we can replace this with a try catch block since the documentation for prisma says the delete() function throws a record not found exception for records that don't exist